.github/workflows/release-chainhook-event-observer.yaml

```diff
@@ -0,0 +1,48 @@
+name: Build Release for Chainhook 
+on:
+  pull_request:
+  push:
+    tags:
+     - 'v*'
+    paths-ignore:
+     - '**/CHANGELOG.md'
+     - '**/package.json'
+     - '**/package-lock.json'
+    paths:
+     - components/chainhook-event-observer
```

Comment:
Same feedback here, we'll need to add a few additional paths:
- components/clarity-repl
- components/clarity-lsp
- components/clarinet-files
- components/clarinet-deployments
- components/chainhook-types-rs